x-pack/test/functional/page_objects/lens_page.ts

```diff
@@ -127,7 +132,9 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
       await testSubjects.click('lns-newBucket-add');
       const queryInput = await testSubjects.find('indexPattern-filters-queryStringInput');
       await queryInput.type(queryString);
-      await PageObjects.common.pressEnterKey();
+      // Press Tab 2 twice instead of Enter to avoid race condition with the dropdown
+      await PageObjects.common.pressTabKey();
+      await PageObjects.common.pressTabKey();
```

Comment:
I don't understand why this fixes the issue, what was the race condition? Can we fix that some other way?

Reply:
@dej611 Are you aware of the `queryInput` service which is used for dashboard tests? I think it might simplify this whole test.

Reply:
I've improved the comment here to explain the context of the bug.
Perhaps some other fix is possible, but the double Tab one is the most consistent from tests, as it safely closes the dropdown without auto-completing the query field.